mlog/log.go

```diff
@@ -28,6 +28,7 @@ type Field = zapcore.Field
 
 var Int64 = zap.Int64
 var Int = zap.Int
+var Uint32 = zap.Uint32
```

Comment:
Entirely unrelated to this PR, but did we ever consider using zap's `SugaredLogger` as a compromise between "faster" and "convenient"?

Reply:
Yup. I considered it, but I don't really see the need to add that extra sugar.